en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+## Analyzing statistics
+
+Once you access the Insights Portal, you can view the following information.
+
+### Overview
+
+This page gives you a quick overview of the whole system. The targeted audiences for the page are managers and business users who need quick insight. Also, this page can be used as a dashboard to view the current system status.
+
+![Overview](../assets/img/insights/overview.png){.cInlineImage-full}
+
+The information displayed is as follows:
+
+- **Total Traffic**
+
+    ![Total traffic](../assets/img/insights/overview-page-total-traffic.png){.cInlineImage-half}
+
+    This widget displays the total traffic your selected environment received during the selected time range. This includes both successful requests and error requests. To investigate further, you can click the arrow icon on the bottom right corner of the widget to open the [**Traffic** page](#traffic)
+    
+- **Error request count**
+
+    ![Error request count](../assets/img/insights/overview-page-error-request-count.png){.cInlineImage-half}
+
+    This widget displays the total number of requests that have resulted in errors in your selected environment during the selected time range.
+    
+- **Average Error Rate**
+
+    ![Average error rate](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}
+
+    This widget displays the average error rate (error count/total request count) in your selected environment for the selected time interval. You can use this widget as an indicator to know the health of the system. If the error rate is high, investigate further using the Viewing API Analytics on Errors. 
+
+- **95th Percentile Latency**
+
+    ![95th percentile latency](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}
+
+    This widget displays the 95th percentile of all API latencies in your selected environment for the selected time interval. You can use this widget to know whether the whole system operates under given SLAs. This metric gives the first indication of slow APIs. To investigate further, see Viewing API Analytics on Latency. 
+
+- **API Request Summary**
+
+    ![API request summary](../assets/img/insights/overview-page-timeline.png){.cInlineImage-full}
+
+    This chart combines the above three widgets and plots them in a timeline. The y-axis on the left displays the request count and the error count. The x-axis shows time, and the y-axis on the right shows the latency in milliseconds. The granularity of the data points is decided based on the time range you have selected. The tooltip provides the exact value of all three metrics accurately.
+
+### Traffic
+
+This page shows information related to the traffic that goes through your API management system. This includes API usage, application usage, resource usage, etc. You can use this page to investigate the usage of APIs and applications, traffic patterns, etc.
```

Comment:
API Management system?

Reply:
When we say applications, what do we mean? in Choreo also we had the word applications. I am wondering if that would cause a confusion.

Reply:
We no longer use the word "applications" for Choreo services and integrations. "Applications" here are the client applications that have subscribed to the APIs.

Reply:
Used the word "API management deployments"